sprinboot-board/src/main/java/com/prgrms/java/dto/user/PostUserRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.prgrms.java.dto.user;
+
+import com.prgrms.java.domain.HobbyType;
+import com.prgrms.java.domain.User;
+
+public record PostUserRequest(UserLoginInfo userLoginInfo, UserSideInfo userSideInfo) {
```

Comment:
```suggestion
public record PostUserRequest(UserLoginInfo userLoginInfo, UserSideInfo userSideInfo) {
```
HTTP 메서드에 의존적인것보다 행위에 맞춰야하지않을까요?
CreateUserRequest

Reply:
그렇군요.. 혹시 HTTP 메서드에 의존하지 않고 행위로 네이밍을 하는 이유가 있을까요?
HTTP 메서드 자체가 행위를 나타내는게 아닌가 하는 궁금증이 듭니다...

Reply:
HTTP 메서드는 충분히 구체적이지 않는 것 같아요. 특히 POST 메서드가 그런것 같습니다. 자원을 생성할 때 혹은 로그인할 때 모두 POST 를 사용하게 될 텐데요. 예를들어 DTO의 이름을 PostXXX 와 같이 작성하게 된다면, 해당 DTO가 어떤 요청에 대한 것인지 알기 힘들 것 같습니다. 따라서 `CreateXXX`, `LoginXXX`와 같이 작성하는 것이 좋다고 생각합니다.

Reply:
Service가 Http, rpc, graphql인지 알 필요가 있을까요?